test/models/user_test.rb

```diff
@@ -1,4 +1,3 @@
-# encoding: UTF-8
```

Comment:
No need to remove this, there are other UTF-8 strings in the file in other tests.